Examples/Scripts/PostFreeSurferPipelineBatch.sh

```diff
@@ -108,7 +107,10 @@ for Subject in $Subjlist ; do
         --freesurferlabels="$FreeSurferLabels" \
         --refmyelinmaps="$ReferenceMyelinMaps" \
         --regname="$RegName" \
-        --use-ind-mean="$UseIndMean"
+        --use-ind-mean="$UseIndMean")
+    
+    echo "Running command: ${job[*]}"
+    "${job[@]}"
```

Comment:
I don't think we want this.  The fsl_sub outputs the job number, which is useful if we realize we made a mistake and need to cancel them, and putting the very long commands in the terminal in between all the job numbers makes that task more difficult.

Reply:
Not sure why this made into this pull request at all.

Reply:
ok